src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/UsherTreeFlow/index.tsx

```diff
@@ -14,14 +15,20 @@ interface Props {
   shouldStartUsherFlow: boolean;
 }
 
+const USHER_DBS: Record<Pathogen, string> = {
+  SC2: "wuhCor1",
+  MPX: "hub_3471181_GCF_014621545.1",
+};
+
 const generateUsherLink = (
   remoteFile: string,
   treeType: string,
-  sampleCount: number
+  sampleCount: number,
+  pathogen: Pathogen
 ) => {
   const encodedFileLink = encodeURIComponent(remoteFile);
 
-  const DB_PARAM = `db=wuhCor1`;
+  const DB_PARAM = USHER_DBS[pathogen];
```

Comment:
I don't believe this will work as-is. The old version was a query param of `db=foobar`, but the `USHER_DBS` lookup is just pulling `foobar` by itself. I assume it still needs to be turned into a query param?

Reply:
ahh I overlooked that, good point. I'll fix it